_pages/openid-connect.md

```diff
@@ -283,12 +283,12 @@ GET https://idp.int.login.gov/openid_connect/userinfo
 Authorization: Bearer hhJES3wcgjI55jzjBvZpNQ
 ```
 
-### User Info Response
+### User info response
```

Comment:
Yeah, @blacktm pointed me at the 18f style guide for this: https://pages.18f.gov/content-guide/capitalization/